apps/documentation/components/packages/design/form/complexForm.tsx

```diff
@@ -74,6 +88,20 @@ export default function Form() {
                 tooltip: 'Your house number',
               },
             },
+            {
```

Comment:
add this code component to field object below that displays the code people can copy to get the data.

Reply:
Sorry, I don't understand!